privacy.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: markdown
+title: Privacy policy
+description: This Privacy Policy describes how we collect, use, and safeguard the information you give us.
+---
+
+This Privacy Policy describes how we collect, use, and safeguard the information you give us. In this agreement, website refers to the Scratch Addons website (https://scratchaddons.com/). Extension refers to our browser extension, which you can download from https://chrome.google.com/webstore/detail/fbeffbjdlemaoicjdapfpikkikjoneco or https://addons.mozilla.org/firefox/addon/scratch-messaging-extension/.
+
+When we talk about the service, we mean any service provided by us, such as the website or our services used in addons. The terms &quot;we,&quot; &quot;us,&quot; and &quot;our&quot; refer to the leaders of the Scratch Addons open source project (https://github.com/ScratchAddons). When you use our services, you agree to this privacy policy.
+
+## Information we collect
+
+We collect a few different kinds of information about you, like what part of the world you are in, what pages you come to our website from, and what device and software you use. We also collect information about how you use our website. We might collect your Scratch username.
```

Comment:
maybe change this ```We also collect information about how you use our website. We might collect your Scratch username.``` to this: ```We also collect your Scratch username (if you have one) and data about how you use ScratchAddons.com```

Reply:
cc @easrng

Reply:
Sounds good, but this isn't my pr anymore.

Reply:
@Explosion-Scratch I think it is "might" because it is an optional thing. Also, don't refer it as ScratchAddons.com. You can refer it as "our website" as stated on the first paragraph.

@easrng Just so you know, you can do a pull request that is directly to the privacy branch.